src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_BALL_VALUE = 101;
+
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(), new Random().nextInt(MAX_BALL_VALUE));
```

Comment:
The same as previous comment.
Do we really need to create a new instance of `ColorSupplier` and `Random` classes every time when this method is called?
It is better to create a new fields as class property.